compilation errors in gen files, if used UInt64 field in http payload.

var _ = Service("calc", func() {
        Description("The calc service performs operations on numbers.")
        Method("add", func() {
            Payload(func() {
	            Field(1, "some_field", UInt64, "some field", func() {
	                Example(10000)
	            })
	        })
	        Result(Int)
	        HTTP(func() {
	            GET("/add")
	            Body(Empty)
		    Param("some_field")
	        })
	    })
	    Files("/openapi.json", "./gen/http/openapi.json")
    })

After running `goa gen calc/design` I am getting following content in file `calc/gen/http/calc/client/cli.go`

    // BuildAddPayload builds the payload for the calc add endpoint from CLI flags.
    func BuildAddPayload(calcAddSomeField string) (*calc.AddPayload, error) {
	    var err error
	    var someField *uint64
	    {
		    if calcAddSomeField != "" {
			    val, err := strconv.ParseUint(calcAddSomeField, 10, 64)
			    someField = &val
			    if err != nil {
				    return nil, fmt.Errorf("invalid value for someField, must be UINT64")
			    }
		    }
	    }
	    payload := &calc.AddPayload{
		    SomeField: someField,
	    }
	    return payload, nil
    }

which would result in:
    ./cli.go:18:6: err declared and not used

I tried changing UInt64 to UInt and it seems to work, as it generates the following code:

    // BuildAddPayload builds the payload for the calc add endpoint from CLI flags.
    func BuildAddPayload(calcAddSomeField string) (*calc.AddPayload, error) {
	    var err error
	    var someField *uint
	    {
		    if calcAddSomeField != "" {
			    var v uint64
			    v, err = strconv.ParseUint(calcAddSomeField, 10, 64)
			    val := uint(v)
			    someField = &val
			    if err != nil {
				    return nil, fmt.Errorf("invalid value for someField, must be UINT")
			    }
		    }
	   }
	   payload := &calc.AddPayload{
		    SomeField: someField,
	    }
	    return payload, nil
    }

--- a/codegen/cli/cli.go
+++ b/codegen/cli/cli.go
@@ -493,14 +493,14 @@ func conversionCode(from, to, typeName string, pointer bool) (string, bool) {
 		checkErr = true
 	case uint64N:
 		parse = fmt.Sprintf("%s, err %s= strconv.ParseUint(%s, 10, 64)", target, decl, from)
-		checkErr = true
+		checkErr = decl == ""
 	case float32N:
 		parse = fmt.Sprintf("var v float64\nv, err = strconv.ParseFloat(%s, 32)", from)
 		cast = fmt.Sprintf("%s %s= float32(v)", target, decl)
 		checkErr = true
 	case float64N:
 		parse = fmt.Sprintf("%s, err %s= strconv.ParseFloat(%s, 64)", target, decl, from)
-		checkErr = true
+		checkErr = decl == ""
 	case stringN:
 		parse = fmt.Sprintf("%s %s= %s", target, decl, from)
 	case bytesN:

